user/views.py

```diff
@@ -1,3 +1,29 @@
-from django.shortcuts import render
+import json, bcrypt, re, jwt
+
+from django.views import View
+from django.http import JsonResponse
+
+from .models import User,History
+from my_settings import SECRET,ALGORITHM
+from .utils import login_decorator
+
+
+
+class Signin(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            if User.objects.all().filter(email=data['email']).exists():
+                if bcrypt.checkpw(data['password'].encode('utf-8'),User.objects.get(email=data['email']).password.encode('utf-8')):
+                    access_token = jwt.encode({'id':User.objects.get(email=data['email']).id}, SECRET, ALGORITHM).decode('utf-8')
+                   
+                    return JsonResponse({"message":"SUCCESS", "TOKEN":access_token}, status=200)
+
+                return JsonResponse({"message":"INVALID_PW"},status=401)
+            else:
+                return JsonResponse({"message":"INVALID_EMAIL"},status=401)
```

Comment:
```suggestion 
            return JsonResponse({"message":"INVALID_EMAIL"},status=401)
```
줄일수 있는 else: 는 최대한 줄여주는게 좋습니다.